app/src/main/java/org/oppia/app/options/OptionsFragment.kt

```diff
@@ -0,0 +1,261 @@
+package org.oppia.app.options
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.Observer
+import androidx.lifecycle.Transformations
+import androidx.preference.Preference
+import androidx.preference.PreferenceFragmentCompat
+import androidx.preference.PreferenceManager
+import org.oppia.app.R
+import org.oppia.app.drawer.KEY_NAVIGATION_PROFILE_ID
+import org.oppia.app.model.AppLanguage
+import org.oppia.app.model.AudioLanguage
+import org.oppia.app.model.Profile
+import org.oppia.app.model.ProfileId
+import org.oppia.app.model.StoryTextSize
+import org.oppia.domain.profile.ProfileManagementController
+import org.oppia.util.data.AsyncResult
+import org.oppia.util.logging.Logger
+import javax.inject.Inject
+
+const val KEY_MESSAGE_STORY_TEXT_SIZE = "Text Size"
```

Comment:
generally we are following this convention: `text_size` or `TEXT_SIZE` but not this one.

Reply:
Done.